src/main/kotlin/sharev/team/adapter/outbound/jpa/TeamJpaAdapter.kt

```
@@ -20,20 +20,20 @@ class TeamJpaAdapter(
     private val teamRepository: TeamRepository,
 ) : SaveTeamPort, LoadTeamPort, QueryTeamPort {
 
-    override fun save(title: String): Team {
+    override fun save(title: String, content: String): Team {
         return try {
-            teamRepository.save(TeamJpaEntity(title = title))
+            teamRepository.save(TeamJpaEntity(title = title, content = content))
                 .toDomainModel()
         } catch (e: DataIntegrityViolationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)